### DISCUSSION TITLE
What's the correct capitalization for the name "htmx"?

### DISCUSSION THREAD

**Discussion by MicaelJarniac** [bigskysoftware/htmx#1548] [Q&A] [answered]
Searching in the codebase, I've noticed the following:
| Capitalization | Count | Notes |
| -- | -- | -- |
| **htmx** | 5447 | |
| **HTMX** | 938 | Seems to be used in error messages |
| **Htmx** | 45 | |

**Comment by codetalcott** [comment, marked as answer]
The name htmx is intended to be lowercase. In the docs (sometimes), and in the Hypermedia book, it is initial capitalized when it starts a sentence. I'm not sure how the all-caps version crept in, but the name is officially htmx.

**Reply by MicaelJarniac** [reply]
Why is it title-case when it starts a sentence? In my opinion, this can be confusing, as htmx is a name, so I'd expect its capitalization to be consistent regardless of where it is on a sentence.

**Reply by codetalcott** [reply]
Right, for me it's a question of which is less confusing. A lowercase name is going to cause confusion somewhere. So, in the book, I recommended not putting the confusion at the start of a sentence.

**Reply by vnzio** [reply]
Keep it lower case
But set it in **bold**
Or use a different font

In LaTeX one could manage this nicely
